CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ section for the next release.
 For more information about this file see also [Keep a Changelog](http://keepachangelog.com/) .
 
 ## [Unreleased]
+- Defined Min_upload_level=3
```

Comment:
This needs more context:

1) outside of this PR no one will have any clue what this is referring to
2) I don't think the captures what this PR is adding to the system

Reply:
Got it. I will update this. Thank you!